packages/neuron-ui/src/contexts/NeuronWallet/wallet.ts

```diff
@@ -1,6 +1,8 @@
-export interface Wallet {
+export interface PlainWallet {
```

Comment:
Based on the properties of this new interface, `PlainWallet` might not be suitable as it doesn't really have wallet related properties. Do you think `WalletIdentity` is a better choice?

If we use `WalletIdentity`, then it becomes natural when we load a list of wallet identities instead of `plain` wallets from `Settings` or other places.

Reply:
updated in https://github.com/nervosnetwork/neuron/pull/376/commits/5dd588058d46e30918522306187e4cd65b4a6b21.